README.md

```diff
@@ -49,20 +49,43 @@ To run the container (which will output the package file in your current directo
 docker run -it --rm -v `pwd`:/data <image_name>
 ```
 
+### Using released packages
+Various pre-built packages targeting several popular operating systems can be found under [Releases](/hubblestack/hubble/releases).
+
+## Getting Started
+Hubble runs as a standalone agent on each server you wish to monitor. There are no masters or minions. To get started, install Hubble using one of the above installation options. Once Hubble is installed, check that everything is working correctly:
+1. Run `hubble test.ping`. This should return true.
+1. Run `hubble hubble.sync`. In a default installation, this will sync the latest version of [hubblestack_data](https://github.com/hubblestack/hubblestack_data) to the agent.
+1. Run `hubble hubble.load`. This loads the synced files into Hubble.
+1. Run `hubble hubble.audit`. You should see the results of the default audit configuration run against the agent.
+
+The next step will likely be to switch to your own top file. Create a file `/srv/salt/hubblestack_data/hubblestack_nova_profiles/top.nova` with the contents:
+```
+nova:
+  '*':
+    - network.ssh
+```
+Now, edit the Hubble configuration file `/etc/hubble/hubble`. Uncomment the following lines:
+```
+# file_roots:
+#   base:
+#     - /srv/salt/hubblestack_data
+```
+With the file_root configuration added:
+1. Run `hubble hubble.sync` to pull the files into Hubble's cache
+1. Run `hubble hubble.load`
+1. Run `hubble hubble.audit`. The agent should now only run the network.ssh profile.
```

Comment:
So these three steps are not actually required. By default, `hubble.audit` will both sync and load before it runs. I think we can remove the first two steps from the docs here.

Reply:
Ah ok great thanks. I wondered if that were the case. I've updated it. Cheers.